src/main/java/Events.java

```diff
@@ -0,0 +1,21 @@
+public class Events extends Task {
+    protected String period;
+
+    public Events(String description, String period) {
+        super(description);
+        this.period = period;
+    }
+
+    public void setPeriod(String period) {
+        this.period = period;
+    }
+
+    public String getPeriod() {
+        return period;
+    }
+
+    @Override
+    public String toString() {
+        return ("[D]" + super.toString() + " (at: " + getPeriod() + ")");
```

Comment:
This method exists inside the `Events` class already. It might be better if you printed `period` instead of calling `getPeriod`. Similar improvements can be made throughout the code.